plugins/jobs/src/js/components/Breadcrumbs.js

```diff
@@ -6,57 +6,59 @@ import Breadcrumb from "#SRC/js/components/Breadcrumb";
 import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
 import PageHeaderBreadcrumbs from "#SRC/js/components/PageHeaderBreadcrumbs";
 
-export default function Breadcrumbs({ item, children, states }) {
-  function getBreadcrumb(item, id = "", name = "") {
-    const isDetailPage = [...item.path, item.name].join(".") === id;
-    const link = isDetailPage ? `/jobs/detail/${id}` : `/jobs/overview/${id}`;
-
+export default function Breadcrumbs({ path, name, children, extra }) {
+  /**
+   * @param  {string} link
+   * @param  {string[]} pathname path including name (if given)
+   * @param  {JSX.Element} [extra=null] extra JSX Components to render
+   *
+   * @return {JSX.Element} Breadcrumb Part Component
+   */
+  function buildJSX(link, pathname, extra = null) {
```

Comment:
We could actually pull the function on the upper level as a component itself. That would allow us to give it more concise name like `Breadcrumb` or `Crumb` or `Partial` :) etc. `buildJSX` name is too abstract. Why building JSX, why not just using JSX? Plus why keeping a function inside a function when we could just extract it?

Reply:
I wouldn't expect something called `pathname` being an array. `pathParts`?

Reply:
re function vs component: fair point 👍 

re pathname: see my explanation above, `pathParts` would be okay-ish, but also not completely spot-on 🤔